keyboards/ymdk/ymd09/keymaps/default/keymap.c

```diff
@@ -1,3 +1,18 @@
+/*
+Copyright 2023 SHVD3x
```

Comment:
You cant really claim copyright ownership on a file you have made zero changes to.

Reply:
This file didnt have copyright label at all. I dont know what should I write here, this is needed for lint check.